kubernetes/start

```diff
@@ -29,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup
+./cleanup || true
```

Comment:
This cleanup will basically `docker ps -a | grep k8s | awk '{print $1}' | xargs docker rm -vf`. I don't think it's a big deal whether it succeeds or not.